mne/io/neuralynx/neuralynx.py

```diff
@@ -0,0 +1,187 @@
+import glob
+import os
+
+import numpy as np
+
+from ..._fiff.meas_info import create_info
+from ..._fiff.utils import _mult_cal_one
+from ...utils import _check_fname, _soft_import, fill_doc, logger, verbose
+from ..base import BaseRaw
+
+
+@fill_doc
+def read_raw_neuralynx(
+    fname, preload=False, verbose=None, exclude_fname_patterns: list = None
```

Comment:
Let's add type hints later. Also `verbose` should always be last
```suggestion
    fname, *, preload=False, exclude_fname_patterns=None, verbose=None
```